ui/src/routes.ts

```diff
@@ -6,6 +6,7 @@ export type RouteID =
   | "credentials"
   | "importSchema"
   | "issueCredential"
+  | "issuedQR"
```

Comment:
This is not expressive. Try something like "issuedCredentialQRCode"

Reply:
do u buy [this](https://github.com/0xPolygonID/sh-id-platform/pull/365/files#r1180662714) @cneuro?

Reply:
Adri fixed this.